model/common/src/icon4py/model/common/interpolation/interpolation_factory.py

```diff
@@ -52,8 +56,22 @@ def __init__(
         self._attrs = metadata
         self._providers: dict[str, factory.FieldProvider] = {}
         self._geometry = geometry_source
+        mean_characteristic_length = math.sqrt(
+            mean_cell_area(constants.EARTH_RADIUS, grid.global_num_cells)
+        )
```

Comment:
I had to recreate this computation for here (using an existing helper function, but still...). `CellParams` has a member function `characteristic_length`/`mean_cell_area` with this value already computed. `IconSavepoint` also has a `mean_cell_area` member that I use in the tests. Does it make sense to add a `mean_cell_area` member to `IconGrid` which I could use to access it directly from `self.grid`?